src/components/Select.tsx

```diff
@@ -155,6 +155,7 @@ const SelectInner = styled.div<{
     ...(placement === 'right' && { right: 0, left: 'auto' }),
     width: width ? 10000 : '100%',
     height: 9999,
```

Comment:
I think this line is also causing some issues. Look at the size of the scroll in the story, it can also happen in the app.